subjects/matrix_multiplication/README.md

```diff
@@ -0,0 +1,53 @@
+## matrix_multiplication
+
+### Instructions
+
+- Define a `struct` named `Matrix` as a tuple of 2 tuples. The nested tuple will contain 2 `i32`s.
```

Comment:
here it will be better style to say "it will contain two" instead of "it will contain 2"